server/jetstream_api.go

```diff
@@ -262,7 +262,22 @@ const JSMaxNameLen = 256
 // TODO(dlc) - Move to more generic location.
 type ApiError struct {
 	Code        int    `json:"code"`
+	Kind        string `json:"kind,omitempty"`
 	Description string `json:"description,omitempty"`
+	URL         string `json:"-"`
+	Help        string `json:"-"`
+}
+
+func (e *ApiError) Error() string {
+	return e.Description + " (" + e.Kind + ")"
```

Comment:
Not sure if we expect for this to be called alot but this generates a bunch of temp objects etc. Might be better to do StringBuilder.

Reply:
not a lot but not that its a uint there we also have to do strconv etc - so I think just using fmt.Sprintf() is going to be best.